src/index.js

```diff
@@ -193,7 +196,7 @@ const extractPublicKey = (peerId, entry) => {
   if (entry.pubKey) {
     let pubKey
     try {
-      pubKey = crypto.keys.unmarshalPublicKey(entry.pubKey)
+      pubKey = crypto.keys.unmarshalPublicKey(Buffer.from(entry.pubKey))
```

Comment:
Aren't we replacing the Buffers everywhere?

Reply:
The underlying crypto libraries do not understand Uint8Arrays and require Buffers :(

But! The user should be able to pass Uint8Arrays to our modules without them exploding.

Reply:
This should be updated to use libp2p-crypto@0.18.0, it does use Uint8Array now